eternalcore-core/src/main/java/com/eternalcode/core/bridge/BridgeManager.java

```diff
@@ -1,30 +1,35 @@
 package com.eternalcode.core.bridge;
 
+import com.eternalcode.core.bridge.placeholderapi.PlaceholderApiExtension;
 import com.eternalcode.core.bridge.placeholderapi.PlaceholderApiReplacer;
 import com.eternalcode.core.placeholder.PlaceholderRegistry;
 import org.bukkit.Server;
+import org.bukkit.plugin.Plugin;
 import org.bukkit.plugin.PluginManager;
 
 import java.util.logging.Logger;
 
 public class BridgeManager {
 
     private final PlaceholderRegistry placeholderRegistry;
-    private final Server server;
+    private final Plugin plugin;
```

Comment:
wut? change this to Server

Reply:
The PlaceholderApiExtension class gets information about e.g. a version, name of plugin from Plugin
Server has a method to get plugin information.